{{cookiecutter.repo_name}}/src/pages/account/delete-account.jsx

```diff
@@ -0,0 +1,145 @@
+import { useState } from 'react';
+import { Form, Field, Formik } from 'formik';
+import { TextField } from 'formik-material-ui';
+import { useSnackbar } from 'notistack';
+
+import {
+    Typography,
+    Button,
+    Container,
+    Avatar,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+
+import AccountPageHeader from 'components/AccountPageHeader';
+import { deleteUser, DeleteUserSchema } from 'models/user';
+import { withAuthRequired } from 'util/withAuth';
+
+const useStyles = makeStyles(theme => ({
+    paper: {
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+        paddingTop: theme.spacing(2),
+        paddingBottom: theme.spacing(4),
+    },
+    grid: {
+        marginBottom: theme.spacing(1),
+    },
+    bottomSpace: {
+        marginBottom: theme.spacing(2),
+    },
+}));
+
+const DeleteAccount = () => {
+    const classes = useStyles();
+    const { enqueueSnackbar } = useSnackbar();
+    const [open, setOpen] = useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    return (
+        <Container className={classes.paper} component="main" maxWidth="xs">
+            <AccountPageHeader>
+                <Avatar>
+                    <LockOutlinedIcon />
+                </Avatar>
+            </AccountPageHeader>
```

Comment:
It looks like we could abstract these account pages more as they're all very similar and could save some re-used code